plugins/linux-v4l2/v4l2-output.c

```diff
@@ -191,8 +191,10 @@ static bool virtualcam_start(void *data)
 
 	for (int i = 0; i < n; i++) {
 		char device[32];
+		char device_suffix[27];
 
-		snprintf(device, 32, "/dev/%s", list[i]->d_name);
+		strncpy(device_suffix, list[i]->d_name, 27);
+		snprintf(device, 32, "/dev/%s", device_suffix);
```

Comment:
Same as the other comment, lets log that you have a malformed entry and continue the loop.

Reply:
Same, I think device name will never go beyond a length of 32.